File: king-admin-java/src/main/java/com/oukingtim/domain/SysMenu.java
package com.oukingtim.domain;

import com.baomidou.mybatisplus.annotations.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Created by oukingtim
 */
@TableName("sys_menu")
@Data
@EqualsAndHashCode(callSuper = false)
public class SysMenu extends BaseModel<SysMenu> {

    private String parentId;
    private String name;
    private String type;
    private String icon;
    private String title;
    private Integer level;
    private Integer order;


}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/domain/SysMenu.kt
package com.oukingtim.domain

import com.baomidou.mybatisplus.annotations.TableName

/**
 * Created by oukingtim
 */
@TableName("sys_menu")
class SysMenu : BaseModel<SysMenu>() {
    val parentId: String? = null
    val name: String? = null
    val type: String? = null
    val icon: String? = null
    val title: String? = null
    val level: Int? = null
    val order: Int? = null
}